Sources/ComposableArchitecture/SwiftUI/Alert.swift

```diff
@@ -213,8 +243,7 @@ extension AlertState: CustomDumpReflectable {
       children: [
         "title": self.title,
         "message": self.message as Any,
-        "primaryButton": self.primaryButton as Any,
-        "secondaryButton": self.secondaryButton as Any,
+        "buttons": self.buttons,
```

Comment:
let's add tests for the custom dump of `AlertState` and `ConfirmationDialogState`